.circleci/config.yml

```diff
@@ -56,6 +56,16 @@ jobs:
   build:
     executor: docker_executor
     steps:
+      - run:
+          name: Publish JetBrains/gradle-grammar-kit-plugin locally
+          # language=sh
+          command: |
+            cd ..
+            git clone "https://github.com/MaciejG604/gradle-grammar-kit-plugin.git"
```

Comment:
The version of `gradle-grammar-kit-plugin` is gonna still be 2021.1.2, and it's still gonna be taken from `gradlePortalPlugin`, not `mavenLocal` :/

Reply:
Change in both your PR to gradle-grammar-kit-plugin, and here in libs.versions.toml